pkg/deploy/generator/const.go

```diff
@@ -0,0 +1,24 @@
+package generator
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// Deployment constants
+const (
+	KeyVaultTagName          = "vault"
+	ClustersKeyVaultTagValue = "clusters"
+	ServiceKeyVaultTagValue  = "service"
+	kvClusterSuffix          = "-cls"
+	kvServiceSuffix          = "-svc"
+)
+
+// Template file constants
+const (
+	FileEnvDevelopment         = "env-development.json"
+	FileRPDevelopementNSG      = "rp-development-nsg.json"
+	FileRPDevelopment          = "rp-development.json"
+	FileRPProductionNSG        = "rp-production-nsg.json"
+	FileRPProduction           = "rp-production.json"
+	FileDatabaseDevelopment    = "databases-development.json"
+	FileRPProductionParameters = "rp-production-parameters.json"
+)
```

Comment:
* since you're defining these, use these in deploy.go too - instead of DeployRpProductionJsonBytes, do Asset(FileRPProduction).
* unexport all the ones that are not used in other packages